New/src/content/en/events/embeds/event-map.css

```diff
@@ -0,0 +1,44 @@
+html, body {
```

Comment:
now that it's in the frame box and gets styles from devsite, does it still need all the html,body classes?